jekyll/_cci2/upgrading.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: classic-docs
+title: "Upgrading from CircleCI 1.0 Enterprise Installation to 2.0"
+category: [administration]
+order: 11
+description: "How to upgrade from CircleCI 1.0 Enterprise to CircleCI 2.0"
+---
+
+This document provides instructions for System Administrators upgrading an existing CircleCI 1.0 Enterprise installation to CircleCI 2.0 in the following sections: 
+
+* TOC
+{:toc}
+
+**Notes:** 
+- CircleCI 2.0 is only available on AWS. This installation only supports AWS instance types with attached SSD storage (M3, C3, or R3). EBS-only volumes (C4 or M4) will not work with an installation managed by Terraform. 
```

Comment:
The instance store requirement should only apply to the 1.0 builders (*_builders ASG). Pretty sure 2.0 builders are fine with EBS-only.

Reply:
I removed this note, since it was basically copied from the installation doc. However, do we need to update that doc to make it more accurate?

Reply:
"only available on AWS" - that doesn't fit very well the "The power of CircleCI behind your firewall or in the cloud." shown on the website...?